code/datums/status_effects/debuffs.dm

```diff
@@ -33,11 +33,12 @@
 	ADD_TRAIT(owner, TRAIT_INCAPACITATED, TRAIT_STATUS_EFFECT(id))
 	ADD_TRAIT(owner, TRAIT_IMMOBILIZED, TRAIT_STATUS_EFFECT(id))
 	ADD_TRAIT(owner, TRAIT_HANDS_BLOCKED, TRAIT_STATUS_EFFECT(id))
-
+	owner.add_status_indicator("stunned") // SKYRAT EDIT ADDITION: Status indicators
```

Comment:
As in these instances

Reply:
I don't understand what you mean by use overrides

Reply:
See what I sent you in discord